HLTrigger/Configuration/python/HLT_75e33/modules/hltTiclTrackstersCLUE3DHighL1Seeded_cfi.py

```diff
@@ -100,34 +94,45 @@
       2,
       2
     ),
-    eid_input_name = cms.string('input'),
-    eid_output_name_energy = cms.string('output/regressed_energy'),
-    eid_output_name_id = cms.string('output/id_probabilities'),
-    eid_min_cluster_energy = cms.double(1),
-    eid_n_layers = cms.int32(50),
-    eid_n_clusters = cms.int32(10),
     computeLocalTime = cms.bool(False),
     doPidCut = cms.bool(True),
     cutHadProb = cms.double(999.),
     type = cms.string('CLUE3D')
-
     ),
     pluginPatternRecognitionByFastJet = cms.PSet(
         algo_verbosity = cms.int32(0),
         antikt_radius = cms.double(0.09),
-        eid_input_name = cms.string('input'),
-        eid_min_cluster_energy = cms.double(1),
-        eid_n_clusters = cms.int32(10),
-        eid_n_layers = cms.int32(50),
-        eid_output_name_energy = cms.string('output/regressed_energy'),
-        eid_output_name_id = cms.string('output/id_probabilities'),
         minNumLayerCluster = cms.int32(5),
         type = cms.string('FastJet')
     ),
+    pluginInferenceAlgoTracksterInferenceByDNN = cms.PSet(
+        algo_verbosity = cms.int32(0),
+        onnxPIDModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
+        onnxEnergyModelPath = cms.FileInPath('RecoHGCal/TICL/data/RecoHGCal-TICL/ticlv4/onnx_models/energy_id_v0.onnx'),
+        inputNames  = cms.string('input:0'),
+        outputNames = cms.string("output/regressed_energy:0", "output/id_probabilities:0"),
+        eid_min_cluster_energy = cms.double(1),
+        eid_n_layers = cms.int32(50),
+        eid_n_clusters = cms.int32(10),
+        doPID = cms.int32(0),
+        doRegression = cms.int32(0),
+        type = cms.string('TracksterInferenceByCNNv4')
```

Comment:
This should be
```suggestion
        type = cms.string('TracksterInferenceByDNN')
```

You want to add to `pluginPatternRecognitionByCLUE3D PSet`  the following entry:
```python
inferenceAlgo = cms.string('TracksterInferenceByCNNv4'),
```
and to the modifier
```python
ticl_v5.toModify(hltTiclTrackstersCLUE3DHighL1Seeded.inferenceAlgo, type = cms.string('TracksterInferenceByDNN'))
```